packages/compass-aggregations/src/components/pipeline-results-workspace/pipeline-results-list.tsx

```diff
@@ -16,13 +16,14 @@ const containerStyles = css({
 
 const PipelineResultsList: React.FunctionComponent<{
   documents: Document[];
+  allDocsExpanded: boolean;
   view: ResultsViewType;
-}> = ({ documents, view }) => {
-  const listProps: React.ComponentProps<typeof DocumentListView> = useMemo(
+}> = ({ documents, allDocsExpanded, view }) => {
+  const listProps = useMemo(
     () => ({
       docs: documents.map((doc) => new HadronDocument(doc)),
       isEditable: false,
-      copyToClipboard(doc) {
+      copyToClipboard(doc: HadronDocument) {
         const str = doc.toEJSON();
         void navigator.clipboard.writeText(str);
       },
```

Comment:
Actually while we are at it, what's the reason to keep these in the memo? Both are not changing at all no matter what happens to the document list, right? 🤔

Reply:
currently it will just avoid mapping of docs, when `isExpanded` changes. i'd leave it as is.